src/style.css

```diff
@@ -0,0 +1,91 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.form {
+  display: flex;
+  margin: 20px auto;
+  width: 100%;
+  height: 70px;
+}
+
+::placeholder {
+  font-family: "Avenir", sans-serif;
+  color: #3d4e61;
+  font-weight: 300;
+}
+
+.form--small {
+  width: 100%;
+  height: 42px;
+}
+
+.label_input {
+  display: flex;
+  width: 100%;
+}
+
+.icon {
+  width: 19px;
+  position: absolute;
+  margin-top: 25px;
+  margin-left: 26px;
+}
+
+.icon--small {
+  width: 11px;
+  position: absolute;
+  margin-top: 15px;
+  margin-left: 13px;
+}
+
+.input {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  color: #3d4e61;
+  outline: none;
+  padding-left: 62px;
+  font-size: 16px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.input--small {
+  padding-left: 33px;
+  padding-top: 0;
+  padding-bottom: 2px;
+  font-size: 14px;
+}
+
+.input:hover {
+  box-shadow: 0 4px 4px rgba(0, 0, 0, 0.15);
+}
+
+.input:focus-within {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 3px;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.input--small:hover {
+  text-shadow: none;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input--small:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}

```

Comment:
this styles almost identical , lets write styles for all inputs on hover & focus(DRY)